scalyr

```diff
@@ -108,7 +145,11 @@ def sendRequest(args, uri, parameterDict):
         print_stderr("Connecting to %s via %s" % (serverAddress, protocol))
 
     conn = None
-    if useSSL:
+    if useSSL and useProxy:
```

Comment:
There should be a case for `useSSL` being false and `useProxy` is true as well right? I assume just something like this:
```
proxyAddress = parseProxyAddress(proxyAddress)
conn = httplib.HTTPConnection(proxyAddress.hostname, proxyAddress.port)
conn.set_tunnel(serverAddress)
```

Reply:
Good point. I can add that back in.